# Pendulum angles
std_msgs/Header header
float64 alpha
float64 beta
float64 g_alpha
float64 g_beta